routes/userRouter.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const router = express.Router();
+const userController = require('../controllers/userController');
+
+// GET: /user/:id (get user)
+router.get('/user/:id', userController.getUser);
+
+// GET : /user/:id/events
+router.get('/user/:id/events', userController.getUserEvents);
+
+// GET: /user/:id/reservations
+router.get('/user/:id/reservations', userController.getUserReservations);
+
+// POST: /newUser
+router.post('/newUser', userController.createUser);
+
+// PATCH: /user/:id (updateUser)
+router.put('/user/:id', userController.updateUser);
```

Comment:
More of a stylistic convention, but usually RESTful routes are plural, so `user` would be `users` .